http-client.ts

```diff
@@ -8,9 +9,10 @@ import filesize = require("filesize");
 export class HttpClient implements Server.IHttpClient {
```

Comment:
httpclient has new behavior as it now throws `Storing credentials is not supported on <platform>` exception. Consider introducing tests for this behavior.